backend2/core/models.py

```diff
@@ -35,3 +35,17 @@ class Follow(models.Model):
     follower = models.ForeignKey(User, related_name='following', on_delete=models.CASCADE)
     following = models.ForeignKey(User, related_name='followers', on_delete=models.CASCADE)
     created_at = models.DateTimeField(auto_now_add=True)
+
+class Profile(models.Model):
+    profile_owner = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Those fields already belong to the Profile model. So, writing the prefix profile to all fields is redundant.